src/sizzle.js

```diff
@@ -717,7 +717,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 	hasCompare = rnative.test( docElem.compareDocumentPosition );
 
 	// Element contains another
-	// Purposefully does not implement inclusive descendent
+	// Purposefully does not implement inclusive descendant
```

Comment:
This comment was arguably correct before, but inarguably bad. Can you just rewrite it to something like "purposefully self-exclusive"?